repository/i.json

```diff
@@ -961,6 +961,19 @@
 				}
 			]
 		},
+		{
+			"name": "Invader Zim Color Scheme",
+			"details": "https://github.com/gnat/sublime-invader-zim",
+			"labels": ["color scheme", "colour scheme", "dark", "theme"],
+			"author": "gnat",
+			"description": "Invader Zim Dark Color Scheme. Similar to: Dracula, SynthWave 84, One Monokai, 80's, Synth, Space, Night, Black, Dark, Neon.",
```

Comment:
- `author` is the same as your GitHub username so you can remove it here.
- Similarly, `description` is typically better managed by setting a description on your GitHub repo. That way you can easily change it any time you want.